src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -102,6 +102,10 @@ public async Task<(Error, string login)> GetLoginByCommit(string repoOwner, stri
             }
             catch (Exception ex)
             {
+                if (ex is AbuseException aex)
+                {
+                    Log.Write($"Failed calling GitHub API '{apiDetail}', message: '{ex.Message}', retryAfterSeconds: '{aex.RetryAfterSeconds}'");
```

Comment:
only applied to GetLoginByCommit? What about GetUserByLogin?

Reply:
Forgot about this... added